.goreleaser.yml

```diff
@@ -97,4 +97,9 @@ snapcrafts:
       386: i386
     apps:
       exercism:
-        plugs: ["home", "network", "removable-media"]
+        plugs: ["home", "network", "removable-media","personal-files"]
+    plugs:
+      personal-files:
+        write:
+        - $HOME/.config/exercism/
+        - $HOME/exercism/
```

Comment:
This is only the default installation directory, and should not be hard coded, i think, but should gather that from the configuration file. Otherwise we are likely going to have intermittent permission problems if the user changes the workspace.

Reply:
We can allow write access to any subdirectory of $HOME - would that be better?

Reply:
I suspect that is better.  Perhaps the configuration file is not yet written at this point in time, to be able to get the workspace from the configuration.